packages/fraction-model/configure/src/model-options.jsx

```diff
@@ -0,0 +1,146 @@
+   * @param {object} event contains event object
+   * @param {string} value contains value of DOM element
+   */
+  change(key, event, value) {
+    const { model, onChange } = this.props;
+    const oldModel = cloneDeep(model);
+    let newModel = cloneDeep(model);
+    let showDiag = false;
+    if (key === 'max') {
+      newModel.maxModelSelected = value;
+      showDiag = true;
+    } else if (key === 'part') {
+      newModel.partsPerModel = value;
+      showDiag = true;
+    } else if (key === 'student-config') {
+      newModel.allowedStudentConfig = value;
+      showDiag = false;
+    }
+    onChange(oldModel, newModel, showDiag);
+  }
+
+  /**
+   * Function to trigger on change of dropdown value model type
+   * @param {string} value contains selection value
+   */
+  handleSelect = (value) => {
+    const { model, onChange } = this.props;
+    model.modelTypeSelected = value?.target.value;
+    onChange(model, { ...model }, false);
+  };
+
+  render() {
+    const { model, classes, modelOptions } = this.props;
+    const { maxOfModel, partsPerModel, modelTypeChoices } = modelOptions;
+
+    return (
+      <div>
+        <CardBar header="Configure Fraction Model"></CardBar>
+        <br />
+        <div className={classes.groupInline}>
+          <div className={classes.group}>
+            <label className={classes.inputLabel}>Model Type</label>
+            <Select className={classes.container2} onChange={this.handleSelect} value={model.modelTypeSelected}>
+              {modelTypeChoices.map((choice, index) => (
+                <MenuItem key={'item_' + index} value={choice.value}>
+                  {choice.label}
+                </MenuItem>
+              ))}
+            </Select>
+          </div>
+          <div className={classes.group}>
+            <label className={classes.inputLabel}>Max # of Models</label>
+            <MiniField
+              min={maxOfModel.min}
+              max={maxOfModel.max}
+              value={model.maxModelSelected}
+              name="max-model"
+              onChange={this.changeMaxModel}
+            />
+          </div>
+          <div className={classes.group}>
+            <label className={classes.inputLabel}>Parts per Model</label>
+            <MiniField
+              min={partsPerModel.min}
+              max={partsPerModel.max}
+              value={model.partsPerModel}
+              name="model-parts"
+              onChange={this.changePartModel}
+            />
+          </div>
+        </div>
+        <div className={classes.checkbox}>
+          <Checkbox onChange={this.studentConfig} checked={model.allowedStudentConfig} label={''} />
+          <span className={classes.chkLabel}>Allow student to configure number of models and parts per model</span>
+        </div>
+      </div>
+    );
+  }
+}
+
+const styles = () => ({
+  groupInline: {
+    alignItems: 'center',
+    display: 'flex',
+    gap: '1.25rem',
+  },
+  group: {
+    margin: '0.75rem 0',
```

Comment:
I suggest using 'px' instead of 'rem'. We had several issues caused by 'rem' and we try to avoid it.

Reply:
Updated